experimental/tf_proxy/server/src/attestation.rs

```diff
@@ -0,0 +1,198 @@
+//
```

Comment:
Is this a copy of oak_functions/attestation ?
Could it reuse the code from there?

Reply:
This is not a straight copy. It needed quite a few modifications, so cannot directly reuse it without significatn refactoring. We can do that in a future PR if we decide to make the proxy more generic.

Reply:
Even if we don't make the proxy more generic, I think we should make it share as much logic as possible with the existing implementation of the attestation library, we cannot afford to maintain two forks. Maybe just break it down into smaller functions and reuse those. (still fine to do in a separate PR of course)

Reply:
Created #2418 to track this.